server/src/util/sendEmail.ts

```diff
@@ -0,0 +1,46 @@
+import * as nodemailer from "nodemailer";
+
+/**
+ * Uses nodemailer to send an email
+ * @param data data used to send the email (from, to, subject, text, html)
+ */
+export default function sendEmail(data: any) {
+
+	// Throw an error if these environment variables aren't defined - we need them
+	const usedEnvVarables = [
+		"EMAIL_HOST", "EMAIL_PORT", "EMAIL_USERNAME", "EMAIL_PASSWORD",
+	];
+
+	usedEnvVarables.forEach((varName) => {
+		if (typeof process.env[varName] === "undefined" || 0 === process.env[varName].length) {
+			throw new Error(`Could not find environment variable: ${varName}`);
+		}
+	});
+
+	const transporter = nodemailer.createTransport({
+		host: process.env["EMAIL_HOST"],
+		port: process.env["EMAIL_PORT"],
+		secure: true,
```

Comment:
To get the email to work with a test email service like https://mailtrap.io I had to set secure to false.  Could you change `secure` to be an environment variable?